Example/Podfile

```diff
@@ -4,7 +4,16 @@ platform :ios, '10.0'
 
 target 'MoEPluginBaseExample' do
   pod 'MoEPluginBase', :path => '../'
-  pod 'MOGeofence'
+  pod 'MoEPluginInbox', :path => '../'
+  pod 'MoEPluginGeofence', :path => '../'
+
+  pod 'MoEngageRichNotification', :git => 'https://github.com/moengage/MoEngage-iOS-Pods.git', :branch => 'framework'
```

Comment:
I would need access to this repo to run and test the changes. `MoEngage-iOS-Pods`. 
Let's request the access on Monday.